content/en/docs/Releases/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title:  "Release"
+weight: 55
+description:  Kyverno Version Release Notes.
+---
+
+## Kyverno v1.4.2
```

Comment:
As mentioned, there are really quite a lot of other bullet points to mention for each release. Not sure if you intended to cover those in a separate PR at a later time though.

Reply:
This PR was intended to get more feedback regards what the release note should look like.

Reply:
I am currently using the sample you provided to update this Document.